spec/index.html

```diff
@@ -242,7 +242,7 @@ <h2>Notation and Terminology</h2>
       The statement "A = B" means that there is one entity which both expressions "A" and "B" denote. 
       Angle brackets &lt; x, y &gt; are used to indicate an ordered pair of x and y.</p>
 
-    <p>Throughout this document, <a>RDF graph</a>s and other fragments of RDF abstract syntax
+    <p>Throughout this document, <a>RDF graph</a>s and their components
```

Comment:
```suggestion
    <p>Throughout this document, <a>RDF graphs</a> and their components
```

Reply:
Generally speaking, singular terms have plural forms (e.g., term, terms), and plural terms have singular forms (e.g., errata, erratum), which are typically treated as synonymous in ReSpec, with links back to the same definition/description. Part of the justification for this is that the markup and highlight read *much* more clearly and attractively when the full form is within the tag, whether the plural is regular and just adds an "s" (or "es" or similar) (and is usually handled automatically by ReSpec) or irregular and substantially changes the word or its end (and may require explicit aliasing in the ReSpec markup). So no, to my mind, it does not make more sense for the markup to be `<a>RDF graph</a>s and`; rather, it makes more sense for the markup to be `<a>RDF graphs</a> and`.